src/write_http.c

```diff
@@ -507,6 +569,8 @@ static int config_set_format (wh_callback_t *cb, /* {{{ */
                 cb->format = WH_FORMAT_COMMAND;
         else if (strcasecmp ("JSON", string) == 0)
                 cb->format = WH_FORMAT_JSON;
+        else if (strcasecmp ("KAIROSDB", string) == 0)
```

Comment:
Please add this format option to the *collectd.conf(5)* manpage. It is generated from `src/collectd.conf.pod`.